src/styles/stars.css

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(/src/images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+  &--4 .stars__star:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+  }
```

Comment:
This selector implies a BEM naming convention, but the modifier `--4` does not clearly indicate its purpose. The name should describe its function or the state it represents, such as `--4-stars` to indicate that it applies to a 4-star rating.